tests/integration/llm/client.py

```diff
@@ -76,7 +76,19 @@
     "t5-small": {
         "batch_size": [1, 2],
         "max_memory_per_gpu": 2
-    }
+    },
+    "gpt2-xl": {
+        "batch_size": [1, 2],
+        "max_memory_per_gpu": 7.0
+    },
+    "facebook/opt-13b": {
+        "batch_size": [1, 2],
+        "max_memory_per_gpu": 15.0
+    },
+    "bigscience/bloom-7b1": {
+        "batch_size": [1, 2],
+        "max_memory_per_gpu": 17.0
```

Comment:
please use fp16 for testing. Also the artifact saving in this as well. Ideally, we are trying to keep a GPU memory usage under 50% (12GB).

Reply:
makes sense, these tests are written when only fp32 support was tested/present. Before updating these tests, we need to update the wheel with latest changes I guess.

Reply:
updated.